charts/gardener-extension-shoot-cert-service/templates/deployment.yaml

```diff
@@ -62,6 +70,18 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- if .Values.gardenerCertificates.runtimeCluster.virtualKubeAPIServerIncludePrimaryDomain }}
+        - name: VIRTUAL_KUBE_API_SERVER_SNI_INCLUDE_PRIMARY_DOMAIN
+          value: "true"
+        {{- end }}
+        {{- if .Values.gardenerCertificates.seed.enabled }}
+        - name: SEED_NAME
+          value: {{ .Values.gardener.seed.name }}
+        - name: SEED_INGRESS_DNS_DOMAIN
+          value: {{ .Values.gardener.seed.ingressDomain }}
+        - name: SEED_DNS_SECRET_ROLE
+          value: {{ .Values.gardenerCertificates.seed.dnsSecretRole }}
+        {{- end }}
```

Comment:
Shouldn't this information directly be read from the seed object?

Reply:
The seed ingress domain is passed as value from by the `gardenlet` on deployment of the extension. Why should that not be sufficient?